src/components/app.js

```diff
@@ -0,0 +1,8 @@
+import React, { Component } from 'react';
+import WayFinder from './wayfinder/wayfinder.js';
+
+export default class App extends Component {
+    render() {
+        return <WayFinder />;
```

Comment:
I think wayfinder is one word and does not need CamelCasing

Reply:
I will make the quick change now! 👍